core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -56,122 +56,123 @@ var ExplorationEditorFeedbackTab = function() {
   /*
    * Workflows
    */
-  this.acceptSuggestion = function(suggestionDescription) {
-    return element.all(by.css(suggestionRowClassName)).then(function(rows) {
-      var matchingSuggestionRows = rows.filter(function() {
-        return explorationFeedbackSubject.getText().then(function(subject) {
-          return suggestionDescription.indexOf(subject) !== -1;
-        });
-      });
-      expect(matchingSuggestionRows[1].isDisplayed()).toBe(true);
-      matchingSuggestionRows[1].click();
-      expect(viewSuggestionButton.isDisplayed()).toBe(true);
-      viewSuggestionButton.click();
-      expect(acceptSuggestionButton.isDisplayed()).toBe(true);
-      suggestionCommitMessageInput.sendKeys('Commit message');
-      acceptSuggestionButton.click();
-      waitFor.invisibilityOf(
-        acceptSuggestionButton, 'Suggestion modal takes too long to disappear');
-      waitFor.pageToFullyLoad();
-    });
+  this.acceptSuggestion = async function(suggestionDescription) {
+    var rows = await element.all(by.css(suggestionRowClassName));
+    var matchingSuggestionRows = [];
+    for (var i = 0; i < rows.length; i++) {
```

Comment:
While array methods can usually be called on `ElementArrayFinder` objects, this doesn't work until after the matching elements have been found. I think it's safer to do `await rows.count()`